src/crud/peer.py

```
@@ -0,0 +1,193 @@
+
+async def get_peers(
+    workspace_name: str,
+    filters: dict[str, str] | None = None,
+) -> Select[tuple[models.Peer]]:
+    stmt = select(models.Peer).where(models.Peer.workspace_name == workspace_name)
+
+    stmt = apply_filter(stmt, models.Peer, filters)
+
+    stmt = stmt.order_by(models.Peer.created_at)
+
+    return stmt
```

Ruff (0.12.2)

Unnecessary assignment to `stmt` before `return` statement

Remove unnecessary assignment

(RET504)